src/Our.Umbraco.Vorto/Web/Controllers/VortoApiController.cs

```diff
@@ -46,6 +46,7 @@ public object GetDataTypeByAlias(string contentType, string contentTypeAlias, st
                 case "member":
                     ct = Services.MemberTypeService.Get(contentTypeAlias);
                     break;
+                case "settings":
```

Comment:
Hmm, I think the only way you could do it is to detect it in the angular code (seeing what the URL of the page your are on is and if it's a type editor, what entity type it is) and then pass that in instead of the current "settings" value on this line: https://github.com/djscorch/umbraco-vorto/blob/f1e44d7920b8a9fb441e37e5661522ad59b22785/src/Our.Umbraco.Vorto/Web/UI/App_Plugins/Vorto/js/vorto.js#L314